project/plugins.sbt

```diff
@@ -1,9 +1,7 @@
+addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.14.1")
+addSbtPlugin("io.spray" % "sbt-revolver" % "0.7.2")
+addSbtPlugin("org.scalastyle" %% "scalastyle-sbt-plugin" % "0.7.0")
```

Comment:
Brought over from backend's plugins. My (perhaps incorrect) understanding is that sbt doesn't honor subproject plugins.sbt